kedro-datasets/RELEASE.md

```diff
@@ -1,10 +1,13 @@
 # Upcoming Release
 ## Major features and improvements
+
 ## Bug fixes and other changes
 
+- Fix polars.CSVDataset `save` method on Windows.
```

Comment:
Do you want to mention that made the default encoding for the dataset UTF-8, fixing it?